rpc/common/moc_common_common.proto

```diff
@@ -12,6 +12,8 @@ enum Operation {
 	POST = 1;
 	DELETE = 2;
 	UPDATE = 3;
+	IMPORT = 4;
```

Comment:
can we try to reuse Create for Import.

Reply:
IMO, the create and import operations, as they are implemented right now are different enough that they should require separate APIs. The create operation does not allow caller to specify the key whereas the import operation allows caller to specify it. Merging them together will remove the possibility of controlling access to specific operation using RBAC. 
The approach of keeping import and create as separate operations is consistent with what Azure Key Vault does today:
![image](https://user-images.githubusercontent.com/92179261/167459783-dfec52a3-a94a-46f2-b8e7-7736f9c9ec9c.png)